package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "systemjs",
   "version": "6.2.6",
+  "main": "dist/system-node.js",
```

Comment:
Alternatively, we could change this to `src/system-node.js` and require all users to use a new enough version of node to support NodeJS ES modules.

Reply:
`dist/system-node.js` seems good to me. Unfortunately while Node.js modules still have the experimental warning it's not really something we can do just yet.